src/org/zaproxy/zap/extension/pscanrulesAlpha/XPoweredByHeaderInfoLeakScanner.java

```diff
@@ -71,7 +71,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 		    				"", // Other info
 		    				getSolution(), //Solution
 		    				getReference(), //References
-		    				xpbDirective,	// Evidence - Return the X-Powered-By Header info
+		    				"X-Powered-By: " + xpbDirective, // Evidence - Return the X-Powered-By Header info

```

Comment:
It could be used getHeadersAsString() to extract the (whole) header (with a regular expression?). That should be a perfect match when highlighting.

Yeah, the header name should be moved to a constant.